app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[dict[str, str | int | None]]) -> list:
+    person_list = []
+    [
+        person_list.append(Person(human["name"], human["age"]))
+        for human in people
+    ]
+    for index_person, person in enumerate(people):
+        if "wife" in person and person.get("wife") is not None:
```

Comment:
```suggestion
        if person.get("wife") is not None:
```
You can simplify this